recipes/conda-env/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "2.4.5" %}
+
+package:
+  name: conda-env
+  version: {{ version }}
+
+source:
+  fn: conda-env-{{ version }}.tar.gz
+  url: https://github.com/conda/conda-env/archive/v{{ version }}.tar.gz
+  sha256: 5ce2dd1b023a84ee76d19d508e519c1ca84010a60744976aa3caa42ba7ab38f9
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
Does this need an entry point?
```
  entry_points:
    - conda-env = conda_env.cli.main:main
```

The [anaconda-recipes](https://github.com/ContinuumIO/anaconda-recipes/blob/master/conda-env/meta.yaml) has it but not the repositories recipe